app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -1,11 +1,11 @@
 package com.example.henzoshimada.feeltrip;
 
-import android.location.Location;
```

Comment:
I believe with this implementation Elasticsearch can implicitly handle Doubles as well, so we shouldn't have loss of accuracy.